telos-backend/src/routes/api/calendar.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/:date/:position', (req, res) => {
```

Comment:
Just a thought: Would it be better to use an id parameter here ?(when using mongoose, theres always an implicit _id field set up I believe) The position might be stored as a tuple, we might need to do some string parsing to find the right object. But this definitely works, just depends on what we want front-end to be passing around.

Reply:
True, it might be better to use the implicit id. I was just basing this off the schema we drafted up in the backend x frontend meeting minutes.

Reply:
I agree. I don't think we should be using these kind of parameters to get the widgets. It would cause many problems if we ever change the type of parameters (also what if you have two of the same widgets with the same positions on top of each other?). Using the widget id would be more appropriate.

Reply:
Done. I've changed it so we use the widget id as the path param now.